main.py

```diff
@@ -126,34 +126,20 @@ def check_password(self, email, password):  #Returns 0 for all good, returns 1 f
 
         return {}  # success
 
-class RegisterHandler (BaseHandler):
-    def post(self):
-        email = self.request.get('email')
-        password = self.request.get('password')
-
-        if email[-17:] != "@eastsideprep.org":
-            self.response.write(json.dumps(ERR_SIGNUP_EMAIL_NOT_EPS))
-            return
-
-        if not self.check_signed_up(email):
-            self.response.write(json.dumps(ERR_EMAIL_ALREADY_REGISTERED))
-            return
-
-        self.response.write(json.dumps(REGISTER_SUCCESS))
-        hashed = bcrypt.hashpw(password, bcrypt.gensalt(1))
-        user_obj = User(email = email, password = hashed, verified = False)
-        user_obj.join_date = datetime.datetime.now()
-        db.put(user_obj)
-        row_id = str(user_obj.key().id())
-        logging.info("row id = " + row_id)
-        self.send_confirmation_email(email, row_id)
-
     def check_signed_up(self, email):           #Returns false if there is already a registered user signed up, returns true if there is not
         user_obj_query = db.GqlQuery("SELECT * FROM User WHERE email = :1 AND verified = TRUE", email)
         for query_result in user_obj_query:
             return False
         return True
 
+ERR_NO_ACCOUNT_TO_SEND = {
```

Comment:
I don't think you need an actionable error here. As mentioned before, this should never happen, so a simple error message should be sufficient.